.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -669,6 +669,32 @@ popd > /dev/null
 chown -R ${DAEMON_USER}:${COURSE_BUILDERS_GROUP} ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools
 chmod -R 555 ${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisTools
 
+################################################################################################################
+################################################################################################################
+# BUILD AND INSTALL ANALYSIS TOOLS TS
+
+echo -e "Build and install analysis tools ts"
+
+mkdir -p "${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS"
+
+ANALYSIS_TOOLS_TS_REPO="${SUBMITTY_INSTALL_DIR}/GIT_CHECKOUT/AnalysisToolsTS/"
+
+# Copy cloned files to AnalysisToolsTS directory
+rsync -rtz "${ANALYSIS_TOOLS_TS_REPO}" "${SUBMITTY_INSTALL_DIR}/SubmittyAnalysisToolsTS"
```

Comment:
Is there a reason why we're copying into `SubmittyAnalysisToolsTS` instead of `AnalysisToolsTS`?

Reply:
I was replicating the `AnalysisTools` behaviour. AnalysisTools was copying some files to SubmittyAnalysisTools. https://github.com/Submitty/Submitty/blob/main/.setup/INSTALL_SUBMITTY_HELPER.sh#L636
Should it be changed.

Reply:
I suppose that would be a question for Barb.  Personally, I think `AnalysisToolsTS` is already getting long, and the relation to Submitty is implied, but perhaps Barb is of a different opinion.

Reply:
Let's leave it for now.  We can revisit the name when we eventually get rid of the original AnalysisTools